invenio_rdm_records/resources/config.py

```diff
@@ -357,7 +358,23 @@ class RDMDraftMediaFilesResourceConfig(FileResourceConfig, ConfiguratorMixin):
     {
         LookupError: create_error_handler(
             HTTPJSONException(code=404, description="No grant found with the given ID.")
-        )
+        ),
+        GrantExistsError: create_error_handler(
+            lambda e: HTTPJSONException(
+                code=400,
+                description=e.description,
+            )
+        ),
+    }
+)
+
+user_access_error_handlers = RecordResourceConfig.error_handlers.copy()
```

Comment:
shouldn't be deepcopy?

Reply:
What would this influence? Looks like other error handlers also do just a `copy()`

Reply:
It is important to check this, and also the other error handlers